toolkit/.gitignore

```diff
@@ -1,14 +0,0 @@
-/out/
```

Comment:
Removing this `.gitignore` file seems like an unrelated change.

Reply:
Included by mistake. Will send a separate PR.